Sources/Kodable/KodableError.swift

```diff
@@ -1,46 +0,0 @@
-import Foundation
-
-// MARK: - KodableError
-
-/// Custom errors that `ExtendedCodable` throws when parsing
-public enum KodableError: Error {
```

Comment:
I particularly prefer namespaced errors (e.g. `KodableError`) instead of `Error` - to avoid the mess of confusing `Swift.Error` with the internal `Error` when reading the code.
If a new person lands on this project, as they start reviewing the code and stumble upon an `Error`, they won't know whether it's a `Swift.Error` or a custom internal `Error` unless they search the project for the declaration of a potential internal `Error` at a global scope (and happen to find it easily) 😅

Reply:
I'm fine with reverting that, can you do it 🙏🏻 ? @rogerluan would you prefer KodableError or some other variation ?

Reply:
`KodableError` is good IMO. Whoever gets to this first, gets to take it <img width="20" alt="sassyparrot" src="https://user-images.githubusercontent.com/8419048/102775583-e657af80-436b-11eb-943c-36c5110367aa.gif">

Reply:
Done 😉